include/deal.II/lac/solver_gmres.h

```diff
@@ -83,6 +83,17 @@ namespace internal
       VectorType &operator() (const unsigned int i,
                               const VectorType   &temp);
 
+      /**
+       * Returns size of data vector. It is used in the solver to store
+       * the Arnoldi vectors.
+       */
+      inline
```

Comment:
For an in-class definition of a method, you do not need to put the 'inline' keyword. It will automatically be inlined.